### PR TITLE
Add handler for Philips LTD011 downlight

### PR DESCRIPTION
**Pull request by joshuaspence**
This is mostly so that I can control the on-off behaviour for these lights.

### CODE REVIEW

**Comment by Adminiuga** [comment]
Could you run black and isort on your changes

**Comment by joshuaspence** [comment]
@Adminiuga Done

**Comment by joshuaspence** [comment]
Hmm, not sure what happened as I used `pre-commit`. Will check tomorrow

**Comment by joshuaspence** [comment]
Ok, builds should pass now. Sorry about that.

File: zhaquirks/philips/ltd011.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                DEVICE_TYPE: zha.DeviceType.COLOR_TEMPERATURE_LIGHT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    PhilipsOnOffCluster,
+                    PhilipsLevelControlCluster,
+                    LightLink.cluster_id,
+                    768,
```
Comment (Adminiuga) [inline zhaquirks/philips/ltd011.py:82]: this should be
```suggestion
                    Color.cluster_id,
```
but import it at top with `from zigpy.zcl.clusters.lighting import Color` see https://github.com/zigpy/zha-device-handlers/blob/66cdaf32eeb744f44453f5fca057c3c9ec42f2ac/zhaquirks/philips/zllextendedcolorlight.py#L14 for example

File: zhaquirks/philips/ltd011.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                    4096,
```
Comment (Adminiuga) [inline zhaquirks/philips/ltd011.py:83]: this should be `LightLink.cluster_id`

**Review by Adminiuga — CHANGES_REQUESTED** [review]
Overall looks good. Make the changes outlined in comments and it is good to go

**Review by Adminiuga — APPROVED** [review]

**Pull request merged**